rest-api-spec/src/main/resources/rest-api-spec/api/indices.get.json

```diff
@@ -4,17 +4,12 @@
     "methods":[ "GET" ],
     "url":{
       "path":"/{index}",
-      "paths":[ "/{index}", "/{index}/{feature}" ],
```

Comment:
Interestingly enough, we already do have these endpoints in the spec, they just weren't in this file :)

See: https://github.com/dakrone/elasticsearch/blob/remove-comma-parsing-of-features/rest-api-spec/src/main/resources/rest-api-spec/api/indices.get_mapping.json#L7